src/db/nedb.js

```diff
@@ -57,13 +63,26 @@ class DBHelper {
     }
   }
 
-  static async insertOrUpdateCOracle(db, oracle) {
+  static async removeTopic(db, txid) {
```

Comment:
nit: I think we should use `db.Topic` instead of `db`. The term `db` stands of different entities in different func

Reply:
renamed to topicDb for explicitness

Reply:
also changed for removeOracle